spyder/plugins/help/tests/test_plugin.py

```diff
@@ -30,29 +28,21 @@
 # Fixtures
 # =============================================================================
 @pytest.fixture
-def help_plugin(qtbot):
+def help_widget(qtbot, mocker):
```

Comment:
A couple of questions about this?

* How did you find that it was necessary to test `HelpWidget` and not `Help`, the plugin?
* This file is not testing the plugin anymore, so it should be renamed to `test_main_widget.py`. However, I'd really like to keep testing the plugin so we're sure to not break its API in the future. Is that not possible at all?

Reply:
> How did you find that it was necessary to test `HelpWidget` and not `Help`, the plugin?

The old code attempted set an instance of `Help` as the central widget of a `QMainWindow`. This failed because `Help` is not a `QWidget`.

> However, I'd really like to keep testing the plugin so we're sure to not break its API in the future. Is that not possible at all?

Mhh, looking at the classes `Help` derives from, this should be possible. I’ll update the PR tomorrow.

Reply:
I have made the requested changes. Note that the test `test_help_opens_when_show_tutorial_unit` is now completely rewritten. With the test being disabled/unmaintained for quite some time, I think that the old version is not applicable anymore.